### PR TITLE
chore(app): rename *AppComponent to just AppComponent

### PR DESCRIPTION
**Pull request by filipesilva**
@Brocco can you review?

### CODE REVIEW

File: addon/ng2/blueprints/ng2/files/__path__/app/app.component.ts [hunk 1]
```diff
@@ -3,16 +3,16 @@ import { APP_SHELL_DIRECTIVES } from '@angular/app-shell';<% } %>
 
 @Component({
   moduleId: module.id,
-  selector: '<%= htmlComponentName %>-app',
+  selector: 'app-root',
```
Comment (Brocco) [inline addon/ng2/blueprints/ng2/files/__path__/app/app.component.ts]: "app" should be based upon the value of the prefix coming from angular-cli.json

File: addon/ng2/blueprints/ng2/files/__path__/index.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
   <% } %>
 </head>
 <body>
-  <<%= htmlComponentName %>-app>Loading...</<%= htmlComponentName %>-app>
+  <app-root>Loading...</app-root>
```
Comment (Brocco) [inline addon/ng2/blueprints/ng2/files/__path__/index.html]: "app" should be based upon the value of the prefix coming from angular-cli.json

File: addon/ng2/blueprints/ng2/files/__path__/app/app.component.ts (hunk 1 above)
Comment (filipesilva) [inline addon/ng2/blueprints/ng2/files/__path__/app/app.component.ts]: Done

File: addon/ng2/blueprints/ng2/files/__path__/index.html (hunk 2 above)
Comment (filipesilva) [inline addon/ng2/blueprints/ng2/files/__path__/index.html]: Done

**Pull request merged**